pkg/controller/controller_binding.go

```diff
@@ -396,11 +433,7 @@ func (c *controller) reconcileServiceInstanceCredential(binding *v1alpha1.Servic
 					return err
 				}
 
-				// TODO: We need to delete the ServiceInstanceCredential from the
-				// Broker since the Bind request was successful. This needs to be
-				// addressed as part of orphan mitigiation.
-
-				return nil
+				return c.setOrphanMitigation(binding, toUpdate, instance, serviceClass, brokerName, "too much time has elapsed")
```

Comment:
There is an uncovered scenario surrounding injecting the credential where we might leave an orphan binding. Let's say that the Bind request was successful, but the credential injecting failed. Then, let's say that all subsequent reconciliations have non-failing errors when attempting to send the Bind request to the Broker. Once the retry duration elapses, the reconciler will give up on the ServiceInstanceCredential but will not enter orphan mitigation, even though the Binding was created on the Broker.